globus_automate_client/cli/flows.py

```diff
@@ -585,7 +585,18 @@ def flow_run(
     ),
     flows_endpoint: str = flows_env_var_option,
     verbose: bool = verbosity_option,
-    output_format: OutputFormat = output_format_option,
+    output_format: ListingOutputFormat = typer.Option(
+        None,
```

Comment:
The first value to typer.Option is what gets used as the default if the command line flag isn't explicitly set. What do you think of setting that to `ListingOutputFormat.json` rather than `None` and updating the `output_format` logic below?

Reply:
If I set a default value then I think it will be impossible to distinguish between the default value and a user's explicit override. Doing it this way allows the user to explicitly watch with JSON or YAML output, which maintains existing functionality:

```
# With a default of None
# ----------------------

globus-automate flow run                # output_format=None so we can pick a default
globus-automate flow run --format json  # output_format="json" so the user explicitly wants JSON


# With a default of "json"
# ------------------------

globus-automate flow run                # output_format="json"
globus-automate flow run --format json  # output_format="json"
```

I don't know if it's truly useful to keep that functionality, though. Should we forbid watching in JSON and YAML formats?

Reply:
Thanks Uriel! 😄